docker/nginx/nginx.conf

```diff
@@ -28,6 +28,22 @@ http {
         gzip_proxied no-cache no-store private expired auth;
         gzip_min_length 1000;
 
+        # Proxy requests to the bucket "wis2box-incoming" to MinIO container running on port 9000
+        # NOTE do not use rewrite, it crashes the upload
+        location /wis2box-incoming/ {
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-Forwarded-Proto $scheme;
+            proxy_set_header Host $http_host;
+
+            proxy_connect_timeout 300;
```

Comment:
Do we need this long of a timeout?

Reply:
not sure, this was taken from the example https://docs.min.io/docs/setup-nginx-proxy-with-minio